web/src/helpers/themes.js

```diff
@@ -12,8 +12,8 @@ export const themes = {
   },
   dark: {
     co2Scale: {
-      steps: [0, 150, 600, 750, 800],
-      colors: ['#2AA364', '#F5EB4D', '#9E4229', '#381D02', '#381D02'],
+      steps: [0, 150, 600, 800, 1100, 1500],
+      colors: ['#2AA364', '#F5EB4D', '#B46C32', '#6B3015', '#2A1602', '#000000'],
```

Comment:
```suggestion
      colors: ['#2AA364', '#F5EB4D', '#9E4229', '#381D02', '#381D02', '#000'],
```
Could you change this to use the old colors for the non color blind themes?